rr_light/run_gui.py

```diff
@@ -0,0 +1,94 @@
+import sys
+
+from PyQt5 import QtWidgets
+
+from rr_light.gui import Ui_MainWindow
+from rr_light.reflection_refraction import (
+    calc_ref_plot,
+    calculate_refr_angl,
+    check_invalid_alfa,
+    check_invalid_index_of_refr,
+    matplotlib_graph,
+)
+
+
+def get_vars(my_app) -> tuple[float, float, float]:
+    """Getting the variables from the user.
+
+    :param self: self
+    :return: Angle of incidence.
+    :return: Index of refraction of the first medium
+    :return: Index of refraction of the second medium
+    """
```

Comment:
You could add in the dostring the exception being raised from `check_invalid_index_of_refr`.